Patch to remove lapacke and lapack from Makefile as that is provided by OpenBlas
J. Sassmannshausen / GSTT
diff --git a/regenie-2.2.4-orig/Makefile b/regenie-2.2.4/Makefile
index 03c56e4..091a7c1 100755
--- a/regenie-2.2.4-orig/Makefile
+++ b/regenie-2.2.4/Makefile
@@ -104,11 +104,11 @@ else ifneq ($(strip $(OPENBLAS_ROOT)),)
   INC        += -I${OPENBLAS_ROOT}/include/
   # static linking
   ifeq ($(strip $(STATIC)),1)
-   SLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -llapack -llapacke -lopenblas
+   SLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -lopenblas
    DLIBS     += -lgfortran
   # dynamic linking
   else
-   DLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -llapack -llapacke -lopenblas -lgfortran
+   DLIBS     += -Wl,-rpath=${OPENBLAS_ROOT}/lib/ -lopenblas -lgfortran
   endif
  endif
 endif
